safehttp/mux.go

```diff
@@ -177,12 +210,12 @@ func (m methodHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}()
 
-	for _, intercep := range m.muxInterceps {
-		intercep.Before(rw, ir)
+	for _, key := range h.intercepsOrder {
```

Comment:
Leaving this as it is for now, but this should be just one slice in the end. The `h.interceps[key]` lookup has some tiny cost and is rather unnecessary. Please note it down for a future PR.